byzcoin/api.go

```diff
@@ -492,8 +508,12 @@ func extractDarcID(sb *skipchain.SkipBlock) (darc.ID, error) {
 		return nil, fmt.Errorf("fail to decode data: %v", err)
 	}
 
-	if len(data.TxResults) != 1 || len(data.TxResults[0].ClientTransaction.Instructions) != 1 {
-		return nil, errors.New("genesis darc tx should only have one instruction")
+	if len(data.TxResults) != 1 {
+		return nil, errors.New("genesis block should only have one transaction")
+	}
+
+	if len(data.TxResults[0].ClientTransaction.Instructions) != 2 {
```

Comment:
This won't work on old byzcoin chains right ?

Reply:
you're right, i didn't notice this one